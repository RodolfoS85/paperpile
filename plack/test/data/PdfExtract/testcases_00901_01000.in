---
file: ../../../PDFREPOSITORY/000901.pdf
---
file: ../../../PDFREPOSITORY/000902.pdf
---
file: ../../../PDFREPOSITORY/000903.pdf
---
file: ../../../PDFREPOSITORY/000904.pdf
---
file: ../../../PDFREPOSITORY/000905.pdf
---
file: ../../../PDFREPOSITORY/000906.pdf
---
file: ../../../PDFREPOSITORY/000907.pdf
---
file: ../../../PDFREPOSITORY/000908.pdf
---
file: ../../../PDFREPOSITORY/000909.pdf
---
file: ../../../PDFREPOSITORY/000910.pdf
---
file: ../../../PDFREPOSITORY/000911.pdf
---
file: ../../../PDFREPOSITORY/000912.pdf
---
file: ../../../PDFREPOSITORY/000913.pdf
---
file: ../../../PDFREPOSITORY/000914.pdf
---
file: ../../../PDFREPOSITORY/000915.pdf
---
file: ../../../PDFREPOSITORY/000916.pdf
---
file: ../../../PDFREPOSITORY/000917.pdf
---
file: ../../../PDFREPOSITORY/000918.pdf
---
file: ../../../PDFREPOSITORY/000919.pdf
---
file: ../../../PDFREPOSITORY/000920.pdf
---
file: ../../../PDFREPOSITORY/000921.pdf
---
file: ../../../PDFREPOSITORY/000922.pdf
---
file: ../../../PDFREPOSITORY/000923.pdf
---
file: ../../../PDFREPOSITORY/000924.pdf
---
file: ../../../PDFREPOSITORY/000925.pdf
---
file: ../../../PDFREPOSITORY/000926.pdf
---
file: ../../../PDFREPOSITORY/000927.pdf
---
file: ../../../PDFREPOSITORY/000928.pdf
---
file: ../../../PDFREPOSITORY/000929.pdf
---
file: ../../../PDFREPOSITORY/000930.pdf
---
file: ../../../PDFREPOSITORY/000931.pdf
---
file: ../../../PDFREPOSITORY/000932.pdf
---
file: ../../../PDFREPOSITORY/000933.pdf
---
file: ../../../PDFREPOSITORY/000934.pdf
---
file: ../../../PDFREPOSITORY/000935.pdf
---
file: ../../../PDFREPOSITORY/000936.pdf
---
file: ../../../PDFREPOSITORY/000937.pdf
---
file: ../../../PDFREPOSITORY/000938.pdf
---
file: ../../../PDFREPOSITORY/000939.pdf
---
file: ../../../PDFREPOSITORY/000940.pdf
---
file: ../../../PDFREPOSITORY/000941.pdf
---
file: ../../../PDFREPOSITORY/000942.pdf
---
file: ../../../PDFREPOSITORY/000943.pdf
---
file: ../../../PDFREPOSITORY/000944.pdf
---
file: ../../../PDFREPOSITORY/000945.pdf
---
file: ../../../PDFREPOSITORY/000946.pdf
---
file: ../../../PDFREPOSITORY/000947.pdf
---
file: ../../../PDFREPOSITORY/000948.pdf
---
file: ../../../PDFREPOSITORY/000949.pdf
---
file: ../../../PDFREPOSITORY/000950.pdf
---
file: ../../../PDFREPOSITORY/000951.pdf
---
file: ../../../PDFREPOSITORY/000952.pdf
---
file: ../../../PDFREPOSITORY/000953.pdf
---
file: ../../../PDFREPOSITORY/000954.pdf
---
file: ../../../PDFREPOSITORY/000955.pdf
---
file: ../../../PDFREPOSITORY/000956.pdf
---
file: ../../../PDFREPOSITORY/000957.pdf
---
file: ../../../PDFREPOSITORY/000958.pdf
---
file: ../../../PDFREPOSITORY/000959.pdf
---
file: ../../../PDFREPOSITORY/000960.pdf
---
file: ../../../PDFREPOSITORY/000961.pdf
---
file: ../../../PDFREPOSITORY/000962.pdf
---
file: ../../../PDFREPOSITORY/000963.pdf
---
file: ../../../PDFREPOSITORY/000964.pdf
---
file: ../../../PDFREPOSITORY/000965.pdf
---
file: ../../../PDFREPOSITORY/000966.pdf
---
file: ../../../PDFREPOSITORY/000967.pdf
---
file: ../../../PDFREPOSITORY/000968.pdf
---
file: ../../../PDFREPOSITORY/000969.pdf
---
file: ../../../PDFREPOSITORY/000970.pdf
---
file: ../../../PDFREPOSITORY/000971.pdf
---
file: ../../../PDFREPOSITORY/000972.pdf
---
file: ../../../PDFREPOSITORY/000973.pdf
---
file: ../../../PDFREPOSITORY/000974.pdf
---
file: ../../../PDFREPOSITORY/000975.pdf
---
file: ../../../PDFREPOSITORY/000976.pdf
---
file: ../../../PDFREPOSITORY/000977.pdf
---
file: ../../../PDFREPOSITORY/000978.pdf
---
file: ../../../PDFREPOSITORY/000979.pdf
---
file: ../../../PDFREPOSITORY/000980.pdf
---
file: ../../../PDFREPOSITORY/000981.pdf
---
file: ../../../PDFREPOSITORY/000982.pdf
---
file: ../../../PDFREPOSITORY/000983.pdf
---
file: ../../../PDFREPOSITORY/000984.pdf
---
file: ../../../PDFREPOSITORY/000985.pdf
---
file: ../../../PDFREPOSITORY/000986.pdf
---
file: ../../../PDFREPOSITORY/000987.pdf
---
file: ../../../PDFREPOSITORY/000988.pdf
---
file: ../../../PDFREPOSITORY/000989.pdf
---
file: ../../../PDFREPOSITORY/000990.pdf
---
file: ../../../PDFREPOSITORY/000991.pdf
---
file: ../../../PDFREPOSITORY/000992.pdf
---
file: ../../../PDFREPOSITORY/000993.pdf
---
file: ../../../PDFREPOSITORY/000994.pdf
---
file: ../../../PDFREPOSITORY/000995.pdf
---
file: ../../../PDFREPOSITORY/000996.pdf
---
file: ../../../PDFREPOSITORY/000997.pdf
---
file: ../../../PDFREPOSITORY/000998.pdf
---
file: ../../../PDFREPOSITORY/000999.pdf
---
file: ../../../PDFREPOSITORY/001000.pdf
