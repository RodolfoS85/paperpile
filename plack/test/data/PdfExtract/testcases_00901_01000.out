---
authors: 'Mattick, John S.'
title: 'RNA regulation: a new genetics?'
---
authors: 'Mattick, John S.'
doi: 10.1371/journal.pgen.1000459
title: The Genetic Signatures of Noncoding RNAs
---
authors: 'Mauduit, Claire and Chatelain, Gilles and Magre, Solange and Brun, Gilbert and Benahmed, Mohamed and Michel, Denis'
title: Regulation by pH of the Alternative Splicing of the Stem Cell Factor Pre-mRNA in the Testis
---
authors: 'Ma, Bin and Wang, Zhuozhi and Zhang, Kaizhong'
title: Alignment between Two Multiple Alignments
---
authors: 'Mayas, Rabiah M and Maita, Hiroshi and Staley, Jonathan P'
doi: 10.1038/nsmb1093
title: Exon ligation is proofread by the DExD/H-box ATPase Prp22p
---
authors: 'Mazur, Alexey K.'
doi: 10.1103/PhysRevE.66.011917
title: Comparative bending dynamics in DNA with and without regularly repeated adenine tracts
---
authors: 'Mccullough, Andrew J. and Berget, Susan M.'
title: G Triplets Located throughout a Class of Small Vertebrate Introns Enforce Intron Borders and Regulate Splice Site Selection
---
authors: 'McDonald, Iain and Smith, Barbara'
title: Partial Symmetry Breaking
---
authors: 'McFail-Isom, Lori and Sines, Chad C and Williams, Loren Dean'
title: 'DNA structure: cations in charge?'
---
authors: 'Mehta, Pankaj and Goyal, Sidhartha and Wingreen, Ned S'
doi: 10.1038/msb.2008.58
title: A quantitative comparison of sRNA-based and protein-based gene regulation
---
authors: 'Meierhans, Daniel and Sieber, Martin and Allemann, Rudolf K.'
title: High affinity binding of MEF-2C correlates with DNA bending
---
authors: 'Mendell, Joshua T and Sharifi, Neda A and Meyers, Jennifer L and Martinez-Murillo, Francisco and Dietz, Harry C'
doi: 10.1038/ng1429
title: Nonsense surveillance regulates expression of diverse classes of mammalian transcripts and mutes genomic noise
---
authors: 'Menke, Matthew and Berger, Bonnie and Cowen, Lenore'
doi: 10.1371/journal.pcbi.0040010
title: 'Matt: Local Flexibility Aids Protein Multiple Structure Alignment'
---
authors: 'Menne, Kerstin M. L. and Hermjakob, Henning and Apweiler, Rolf'
title: A comparison of signal sequence prediction methods using a test set of signal peptides
---
authors: 'Mercer, Tim R. and Dinger, Marcel E. and Mattick, John S.'
title: 'Long non-coding RNAs: insights into functions'
---
authors: 'Merling, Ayelet and Sagaydakova, Nataliya and Haran, Tali E.'
doi: 10.1021/bi020662w
title: A-Tract Polarity Dominate the Curvature in Flanking Sequences
---
authors: 'Messias, Ana C. and Sattler, Michael'
doi: 10.1021/ar030034m
title: Structural Basis of Single-Stranded RNA Recognition
---
authors: 'Meyer, Irmtraud M and Miklós, István'
doi: 10.1186/1471-2199-5-10
title: Co-transcriptional folding is encoded within RNA genes
---
authors: 'Meyer, Irmtraud M. and Miklos, Istvan'
doi: 10.1371/journal.pcbi.0030149
title: 'SimulFold: Simultaneously Inferring RNA Structures Including Pseudoknots, Alignments, and Trees Using a Bayesian MCMC Framework'
---
authors: 'Meyerguz, Leonid and Kleinberg, Jon and Elber, Ron'
doi: 10.1073/pnas.0701393104
title: The network of sequence flow between protein structures
---
authors: 'Mignone, Flavio and Gissi, Carmela and Liuni, Sabino and Pesole, Graziano'
title: Untranslated regions of mRNAs
---
authors: 'Miller, Crispin J. and Attwood, Teresa K.'
title: Bioinformatics goes back to the future
---
authors: 'Miriami, Elana and Sperling, Ruth and Sperling, Joseph and Motro, Uzi'
doi: 10.1261/rna.5112704
title: 'Regulation of splicing: The importance of being translatable'
---
authors: 'Mironov, Andrey A. and Fickett, James Wildon and Gelfand, Mikhail S.'
title: Frequent Alternative Splicing of Human Genes
---
authors: 'Mironov, A. A. and Koonin, E. V. and Roytberg, M. A. and Gelfand, M. S.'
title: Computer analysis of transcription regulatory patterns in completely sequenced bacterial genomes
---
authors: 'Mironov, Andrey A. and Novichkov, Pavel S. and Gelfand, Mikhail S.'
title: 'Pro-Frame: similarity-based gene recognition in eukaryotic DNA sequences with errors'
---
authors: 'Mironov, Andrey A. and Roytberg, Michael A. and Pevzner, Pavel A. and Gelfand, Mikhail S.'
title: Performance-Guarantee Gene Predictions via Spliced Alignment
---
authors: 'Misof, B. and Fleck, G.'
title: 'Comparative analysis of mt LSU rRNA secondary structures of Odonates: structural variability and phylogenetic signal'
---
authors: 'Mistry, Neville and Harrington, Whitney and Lasda, Erika and Wagner, Eric J. and GARCIA-BLANCO-, Mariano A.'
doi: 10.1261/rna.2470903
title: 'Of urchins and men: Evolution of an alternative splicing unit in fibroblast growth factor receptor genes'
---
authors: 'Mitarai, Namiko and Benjamin, Julie-Anna M. and Krishna, Sandeep and Semsey, Szabolcs and Csiszovszki, Zsolt and Masse, Eric and Sneppen, Kim'
doi: 10.1073/pnas.0901466106
title: Dynamic features of gene expression control by small regulatory RNAs
---
authors: 'Mitchell, Philip and Tollervey, David'
title: mRNA stability in eukaryotes
---
authors: 'Mitchell, Philip and Tollervey, David'
title: mRNA turnover
---
authors: 'Mitsouras, Katherine and Wong, Ben and Arayata, Charina and Johnson, Reid C. and Carey, Michael'
doi: 10.1128/MCB.22.12.4390-4401.2002
title: The DNA Architectural Protein HMGB1 Displays Two Distinct Modes of Action That Promote Enhanceosome Assembly
---
authors: 'Miyaso, Hidenobu and Okumura, Masayo and Kondo, Shinichi and Higashide, Satoshi and Miyajima, Hiroshi and Imaizumi, Kazunori'
doi: 10.1074/jbc.M209271200
title: An Intronic Splicing Enhancer Element in Survival Motor Neuron (SMN) Pre-mRNA
---
authors: 'Miyazawa, Sanzo and Jernigan, Robert L.'
title: Self-Consistent Estimation of Inter-Residue Protein Contact Energies Based on an Equilibrium Mixture Approximation of Residues
---
authors: 'Miyazawa, Sanzo'
title: A reliable sequence alignment method based on probabilities of residue correspondences
---
authors: 'Miyazawa, Keisuke and Williams, David A. and Gotoh, Akihiko and Nishimaki, Jiroh and Broxmeyer, Hal E. and Toyama, Keisuke'
title: Membrane-Bound Steel Factor Induces More Persistent Tyrosine Kinase Activation and Longer Life Span of c-kit Gene-Encoded Protein Than Its Soluble Form
---
authors: 'Mizutani, Takaharu and Fujiwara, Toshinobu'
title: 'SBP, SECIS binding protein, binds to the RNA fragment upstream of the Sec UGA codon in glutathione peroxidase mRNA'
---
authors: 'Mjolsness, E. and Castaño, R. and Mann, T. and Roden, J. and Gray, A. and Wold, B.'
title: Clustering Methods for the Analysis of C. elegans Gene Expression Array Data
---
authors: 'Modrek, Barmak and Lee, Christopher J'
title: 'Alternative splicing in the human, mouse and rat genomes is associated with an increased frequency of exon creation and/or loss'
---
authors: 'Modrek, Barmak and Lee, Christopher'
title: A genomic view of alternative splicing
---
authors: 'Modrek, Barmak and Resch, Alissa and Grasso, Catherine and Lee, Christopher'
title: Genome-wide detection of alternative splicing in expressed sequences of human genes
---
authors: 'Mhl, Mathias and Will, Sebastian and Backofen, Rolf'
title: Fixed Parameter Tractable Alignment of RNA Structures Including Arbitrary Pseudoknots
---
authors: 'Mhl, Mathias and Will, Sebastian and Backofen, Rolf'
title: Lifting Prediction to Alignment of RNA Pseudoknots
---
authors: 'Molkentin, Jeffery D. and Black, Brian L. and Martin, James F. and Olson, Eric N.'
title: 'Mutational Analysis of the DNA Binding, Dimerization, and Transcriptional Activation Domains of MEF2C'
---
authors: 'Moll, Isabella and Leitsch, David and Steinhauser, Tanja and Bläsi, Udo'
doi: 10.1038/sj.embor.embor772
title: RNA chaperone activity of the Sm-like Hfq protein
---
authors: 'Moller, Steffen and Croning, Michael D. R. and Apweiler, Rolf'
title: Evaluation of methods for the prediction of membrane spanning regions
---
authors: 'Moniaux, Nicolas and Escande, Fabienne and Batra, Surinder K. and Porchet, Nicole and Laine, Anne and Aubert, Jean-Pierre'
title: Alternative splicing generates a family of putative secreted and membrane-associated MUC4 mucins
---
authors: 'Morgan, Steven R and Higgs, Paul G'
title: Barrier heights between ground states in a model of RNA secondary structure
---
authors: 'Morgenstern, Burkhard and Frech, Kornelie and Dress, Andreas and Werner, Thomas'
title: 'DIALIGN: Finding local similarities by multiple sequence alignment'
---
authors: 'Morin, Steves and Charron, FredericÂ and Robitaille, Lynda and Nemer, Mona'
title: GATA-dependent recruitment of MEF2 proteins to target promoters
---
authors: 'Moses, Alan M and Chiang, Derek Y and Kellis, Manolis and Lander, Eric S and Eisen, Michael B'
title: Position specific variation in the rate of evolution in transcription factor binding sites
---
authors: 'Mosig, Axel and -L. Chen, Julian J. and Stadler, Peter F.'
title: Homology Search with Fragmented Nucleic Acid Sequence Patterns
---
authors: 'Mostert, Volker'
doi: 10.1006/abbi.2000.1735
title: 'Selenoprotein P: Properties, Functions, and Regulation'
---
authors: 'Moult, John and Melamud, Eugene'
title: From fold to function
---
authors: 'Moulton, Vincent and Zuker, Michael and Steel, Michael and Pointon, Robin and Penny, David'
title: Metrics on RNA Secondary Structures
---
authors: 'MucchielliĆGiorgi, M. H. and Hazout, S. and Tufféry, P.'
title: 'PredAcc: prediction of solvent accessibility'
---
authors: 'Muh, Stephanie J. and Hovhannisyan, Ruben H. and Carstens, Russ P.'
doi: 10.1074/jbc.M207409200
title: A Non-sequence-specific Double-stranded RNA Structural Element Regulates Splicing of Two Mutually Exclusive Exons of Fibroblast Growth Factor Receptor 2 (FGFR2)
---
authors: 'Müller-Plathe, Florian'
title: 'Coarse-Graining in Polymer Simulation: From the Atomistic to the Mesoscopic Scale and Back'
---
authors: 'Müller, Christoph W'
title: 'Transcription factors: global and detailed views'
---
authors: 'Mumey, Brendan'
title: 'Probe location in the presence of errors: a problem from DNA mapping'
---
authors: 'Munoz, Victor and Eaton, William A.'
title: A simple model for calculating the kinetics of protein folding from three-dimensional structures
---
authors: 'Munro, Sean'
title: Organelle identity and the targeting of peripheral membrane proteins
---
authors: 'Müntener, Kathrin and Zwicky, Roman and Csucs, Gabor and Baici, Antonio'
doi: 10.1007/s00418-002-0487-y
title: The alternative use of exons 2 and 3 in cathepsin B mRNA controls enzyme trafficking and triggers nuclear fragmentation in human cells
---
authors: 'Murakami, Katsuhiko and Kojima, Toshio and Sakaki, Yoshiyuki'
title: 'Assessment of clusters of transcription factor binding sites in relationship to human promoter, CpG islands and gene expression'
---
authors: 'Muro, Andres F. and Caputi, Massimo and Pariyarath, Rajalakshmi and Pagani, Franco and Buratti, Emanuele and Baralle, Francisco E.'
title: 'Regulation of Fibronectin EDA Exon Alternative Splicing: Possible Role of RNA Secondary Structure for Enhancer Display'
---
authors: 'Murray, Laura J. W. and Bryan Arendall III, W. and Richardson, David C. and Richardson, Jane S.'
doi: 10.1073/pnas.1835769100
title: RNA backbone is rotameric
---
authors: 'Murzin, Alexey G.'
title: Progress in protein structure prediction
---
authors: 'Muthukrishnan, S. and Sahinalp, Sleyman Cenk'
title: Approximate nearest neighbors and sequence comparison with block operations
---
authors: 'Mwangi, Michael M and Siggia, Eric D'
title: Genome wide identification of regulatory motifs in Bacillus subtilis
---
authors: 'Myers, Gene'
title: A Fast Bit-Vector Algorithm for Approximate String Matching Based on Dynamic Programming
---
authors: 'Myers, Eugene W. and Miller, Webb'
title: Optimal alignments in linear space
---
authors: 'Nagaki, Kiyotaka and Cheng, Zhukuan and Ouyang, Shu and Talbert, Paul B and Kim, Mary and Jones, Kristine M and Henikoff, Steven and Buell, C Robin and Jiang, Jiming'
doi: 10.1038/ng1289
title: Sequencing of a rice centromere uncovers active genes
---
authors: 'Nagarajan, Niranjan and Pop, Mihai'
doi: 10.1089/cmb.2009.0005
title: 'Parametric Complexity of Sequence Assembly: Theory and Applications to Next Generation Sequencing'
---
authors: 'Nakaya, Akihiro and Goto, Susumu and Kanehisa, Minoru'
title: Extraction of Correlated Gene Clusters by Multiple Graph Comparison
---
authors: 'Nariai, N. and Kim, S. and Imoto, S. and Miyano, S.'
title: USING PROTEIN-PROTEIN INTERACTIONS FOR REFINING GENE NETWORKS ESTIMATED FROM MICROARRAY DATA BY BAYESIAN NETWORKS
---
authors: 'Nasim, Muhammad Talat and Jaenecke, Susanne and Belduz, Ali and Kollmus, Heike and Flohe, Leopold and McCarthy, John E. G.'
title: Eukaryotic Selenocysteine Incorporation Follows a Nonprocessive Mechanism That Competes with Translational Termination
---
authors: 'Naya, Francisco J. and Wu, Chuanzhen and Richardson, James A. and Overbeek, Paul and Olson, Eric N.'
title: Transcriptional activity of MEF2 during mouse embryogenesis monitored with a MEF2-dependent transgene
---
authors: 'Neduva, Victor and Linding, Rune and Su-Angrand, Isabelle and Stark, Alexander and de Masi, Federico and Gibson, Toby J. and Lewis, Joe and Serrano, Luis and Russell, Robert B.'
doi: 10.1371/journal.pbio.0030405
title: Systematic Discovery of New Recognition Peptides Mediating Protein Interaction Networks
---
authors: 'Neduva, Victor and Russell, Robert B.'
doi: 10.1016/j.febslet.2005.04.005
title: 'Linear motifs: Evolutionary interaction switches'
---
authors: 'Needham, Chris J and Bradford, James R and Bulpitt, Andrew J and Westhead, David R'
title: Inference in Bayesian networks
---
authors: 'Negrini, Massimo and Ferracin, Manuela and Sabbioni, Silvia and Croce, Carlo M.'
doi: 10.1242/jcs.03450
title: 'MicroRNAs in human cancer: from research to therapy'
---
authors: 'Neidle, Stephen'
title: New insights into sequence-dependent DNA structure
---
authors: 'Nekrutenko, Anton and Li, Wen-Hsiung'
title: Transposable elements are found in a large number of human protein-coding genes
---
authors: 'Ner-Gaon, Hadas and Halachmi, Ronit and Savaldi-Goldstein, Sigal and Rubin, Eitan and Ophir, Ron and Fluhr, Robert'
doi: 10.1111/j.1365-313X.2004.02172.x
title: Intron retention is a major phenomenon in alternative splicing in Arabidopsis
---
authors: 'Neves, Guilherme and Zucker, Jacob and Daly, Mark and Chess, Andrew'
doi: 10.1038/ng1299
title: Stochastic yet biased expression of multiple Dscam splice variants by individual cells
---
authors: 'Newton, Anthea and Mackay, Joel and Crossley, Merlin'
doi: 10.1074/jbc.M106256200
title: The N-terminal Zinc Finger of the Erythroid Transcription Factor GATA-1 Binds GATC Motifs in DNA
---
authors: 'AUDREY NGUYEN, M. T. and He, Bin and Karaplis, Andrew C.'
title: Nuclear Forms of Parathyroid Hormone-Related Peptide Are Translated from Non-AUG Start Sites Downstream from the Initiator Methionine
---
authors: 'Nielsen, Henrik and Brunak, Søren and von Heijne, Gunnar'
title: Machine learning approaches for the prediction of signal peptides and other protein sorting signals
---
authors: 'Nielsen, Rasmus and Bustamante, Carlos and Clark, Andrew G. and Glanowski, Stephen and Sackton, Timothy B. and Hubisz, Melissa J. and Fledel-Alon, Adi and Tanenbaum, David M. and Civello, Daniel and White, Thomas J. and Sninsky, John J. and Adams, Mark D. and Cargill, Michele'
doi: 10.1371/journal.pbio.0030170
title: A Scan for Positively Selected Genes in the Genomes of Humans and Chimpanzees
---
authors: 'Nielsen, Henrik and Engelbrecht, Jacob and Brunak, SØREN'
title: A NEURAL NETWORK METHOD FOR IDENTIFICATION OF PROKARYOTIC AND EUKARYOTIC SIGNAL PEPTIDES AND PREDICTION OF THEIR CLEAVAGE SITES
---
authors: 'Nielsen, Henrik and Krogh, Anders'
title: Prediction of signal peptides and signal anchors by a hidden Markov model
---
authors: 'Nielsen, Cydney B. and Shomron, Noam and Sandberg, Rickard and Hornstein, Eran and Kitzman, Jacob and Burge, Christopher B.'
doi: 10.1261/rna.768207
title: Determinants of targeting by endogenous and exogenous microRNAs and siRNAs
---
authors: 'Nikolov, D. B. and Burley, S. K.'
title: 'RNA polymerase II transcription initiation: A structural view'
---
authors: 'Nilsen, Timothy W'
title: Too hot to splice
---
authors: 'Nilsson, Johan and Persson, Bengt and VON HEIJNE, Gunnar'
doi: 10.1110/ps.0226702
title: Prediction of partial membrane protein topologies using a consensus approach
---
authors: 'Ning, Zemin and Cox, Anthony J. and Mullikin, James C.'
doi: 10.1101/gr.194201
title: 'SSAHA: A Fast Search Method for Large DNA Databases'
---
authors: 'Nissen, Poul and Ippolito, Joseph A. and Ban, Nenad and Moore, Peter B. and Steitz, Thomas A.'
doi: 10.1073/pnas.081082398
title: 'RNA tertiary interactions in the large ribosomal subunit: The A-minor motif'
---
authors: 'Noble, William S'
title: What is a support vector machine?
---
authors: 'Nogues, Guadalupe and Kadener, Sebastian and Cramer, Paula and de la Mata, Manuel and Fededa, Juan Pablo and Blaustein, Matias and Srebrow, Anabella and Kornblihtt, Alberto R.'
doi: 10.1080/1521654031000119830
title: 'Control of Alternative Pre-mRNA Splicing by RNA Pol II Elongation: Faster is Not Always Better'
